consultation_analyser/consultations/jinja2/consultations/answers/index.html

```diff
@@ -24,83 +24,6 @@
     ]
   }) }}
 
-<noscript>
```

Comment:
do we want to have some text (even just for the UR testing) along the lines of 'JavaScript is required to view this page', so if people have issues, we'll know what it is?

Reply:
Good point - I'll add in a separate PR.